invoicing_app/management/commands/generate_tax_receipts_new.py

```diff
@@ -19,9 +19,14 @@
 
 from invoicing_app.models import PaymentOptions, Order
 
+from django.db import connection
+from django.db.models.query import ValuesQuerySet
```

Comment:
u use this import?

Reply:
Conection is used in the cursor for the query raw. ValuesQuerySet is not used